.github/workflows/linux.yml

```diff
@@ -143,7 +143,7 @@ jobs:
       - name: Examples STM32F4 Without Discovery Board
         if: always()
         run: |
-          (cd examples && ../tools/scripts/examples_compile.py nucleo_f401re nucleo_f411re nucleo_f429zi nucleo_f446re stm32f411ccu_mini_f401 stm32f411ceu_mini_f411 stm32f407vet6_devebox stm32_f4ve)
+          (cd examples && ../tools/scripts/examples_compile.py nucleo_f401re nucleo_f411re nucleo_f429zi nucleo_f446re black_pill_f401 black_pill_f411 stm32f407vet6_devebox stm32_f4ve)
```

Comment:
Fun: `stm32f411ccu_mini_f401` was never compiled because it wasn't spelled correctly.

Reply:
Pimped the `tools/scripts/examples_check.py` script and a whole host of issues appeared 😱 

```
 $ python3 tools/scripts/examples_check.py

The build path '../build/stm32_f4ve/gui' is outside of the 'modm/build' folder!
 - 'examples/stm32_f4ve/gui/project.xml': '../../../../build/stm32_f4ve/gui'

The build path '../build/arduino_nano/color' is outside of the 'modm/build' folder!
 - 'examples/arduino_nano/color/project.xml': '../../../../build/arduino_nano/color'

Linux CI is missing examples: 'nucleo_f439zi', 'nucleo_f446ze'
    Please add these example folders to the appropriate CI job in
    '.github/workflows/linux.yml'!
```